### PR TITLE
Fix assert typos

### PR DESCRIPTION
**Pull request by xomaczar**

### CODE REVIEW

**Comment by taras** [comment]
Thank you :)

File: addon/helpers/r/get.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const {
 } = Ember;
 
 export function rGet([prop]/*, hash*/) {
-  assert(`r/get expects argument to be not be undefined`, !isNone(prop));
+  assert(`r/get expects argument to be defined`, !isNone(prop));
```
Comment (xomaczar) [inline addon/helpers/r/get.js]: @taras shouldn't this check for presence instead of none? Ember.isPresent

File: addon/helpers/r/get.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ const {
...
 
   return function (target) {
-    assert(`r/get ${prop} expexts object as argument`, typeOf(target) === 'object');
+    assert(`r/get ${prop} expects object as an argument`, typeOf(target) === 'object');
```
Comment (xomaczar) [inline addon/helpers/r/get.js]: I don't think this assert is required Ember.get already checks for this.

Comment (xomaczar) [inline addon/helpers/r/get.js]: Well maybe not. Sorry.

File: addon/helpers/r/get.js (hunk 1 above)
Comment (taras) [inline addon/helpers/r/get.js]: Yes, presence is better.

Comment (xomaczar) [inline addon/helpers/r/get.js]: I'll create a new PR

Comment (taras) [inline addon/helpers/r/get.js]: Thank you :)

**Pull request merged**